### ISSUE TITLE
Incorrect formatting when removing let binding

### ISSUE THREAD

**Issue by siraben** [astro/deadnix#7]
While testing https://github.com/NixOS/nixpkgs/pull/149892, the following file `pkgs/pkgs-lib/tests/default.nix` gets formatted incorrectly:

```nix
# Call nix-build on this file to run all tests in this directory
{ pkgs ? import ../../.. {} }:
let
  formats = import ./formats.nix { inherit pkgs; };
in pkgs.linkFarm "nixpkgs-pkgs-lib-tests" [
  { name = "formats"; path = import ./formats.nix { inherit pkgs; }; }
]
```

deadnix produces
```nix
# Call nix-build on this file to run all tests in this directory
{ pkgs ? import ../../.. {} }:pkgs.linkFarm "nixpkgs-pkgs-lib-tests" [
  { name = "formats"; path = import ./formats.nix { inherit pkgs; }; }
]
```

while maybe something like this is more appropriate
```nix
# Call nix-build on this file to run all tests in this directory
{ pkgs ? import ../../.. {} }:
pkgs.linkFarm "nixpkgs-pkgs-lib-tests" [
  { name = "formats"; path = import ./formats.nix { inherit pkgs; }; }
]
```

**Comment by astro** [comment]
very interesting, thank you for reporting.

**Comment by astro** [comment]
fixed and released in 0.1.2
